CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## Contributing
+
+All pull requests should be made to the develop branch.
+
+```
+git clone https://github.com/vinnymac/PokeNurse
+cd PokeNurse
+git checkout develop
+yarn
```

Comment:
Do you have any preferences on naming conventions for PRs and that of things, @vinnymac? If so we could also add them in here.